pkg/rollout/rollout.go

```diff
@@ -158,7 +184,23 @@ func (r *Rollout) SplitTraffic(svc *run.Service, stable, candidate string) *run.
 	}
 
 	svc.Spec.Traffic = traffic
+	return svc
+}
+
+// Rollback redirects all the traffic to the stable revision.
+func (r *Rollout) Rollback(svc *run.Service, stable string) *run.Service {
+	traffic := []*run.TrafficTarget{newTrafficTarget(stable, 100, StableTag)}
+
+	// Respect tags manually introduced by the user (e.g. UI/gcloud).
+	customTags := userDefinedTrafficTags(svc)
+	traffic = append(traffic, customTags...)
+
+	// Always assign latest tag to the latest revision.
+	traffic = append(traffic, &run.TrafficTarget{LatestRevision: true, Tag: LatestTag})
```

Comment:
these are copypasta from SplitTraffic.
create a utility func like `inheritRevisionTags(svc)` and modify  the svc